policyengine_us/variables/gov/states/la/tax/income/credits/school_readiness/quality_rating_of_child_care_facility.py

```diff
@@ -0,0 +1,10 @@
+from policyengine_us.model_api import *
+
+
+class quality_rating_of_child_care_facility(Variable):
```

Comment:
Since Louisiana-specific
```suggestion
class la_quality_rating_of_child_care_facility(Variable):
```